content/05.use-cases-c.md

```diff
@@ -31,41 +34,45 @@ Often with these projects, once the research contract expires, the research soft
-The use of the "Teams" feature can allow certain groups to have varying levels of access to repos in the organization with a select group having push access to some repos but not others. 
-This can manifest in a group working on some common dataset(s) (e.g. some genetic data) to have push access to the handful of repositories used for processing sequence data, while another group of students/researchers may have push access to an entirely different set of repos. 
+As an example, GitHub Organizations are particularly well-suited to house documents and projects within a laboratory, such as research compendia, codes of conduct, protocols, training documents, and other such documents that evolve collaboratively over time and are relevant to many colleagues.
+In this way, students or teams can have full ownership of repositories within an organization, while ensuring that these materials stay accessible to the laboratory after people have moved on (or upgraded their computers).
+This application extends to research centres, which may include several distinct projects that remain linked under a given institution, such as the German Centre for Integrative Biodiversity Research (iDiv,<https://github.com/idiv-biodiversity>).
+Of course, the utility of this tool goes beyond laboratories - they are useful to structure the organization, presentation, and outcomes of working groups such as the hackathon which inspired this paper (SORTEE-Github-Hackathon, <https://github.com/SORTEE-Github-Hackathon>) by keeping track of all materials as ideas develop and take shape in one virtual space.
+Organizations are also convenient for hosting a set of related learning materials such as a set of lectures or workshops, such as the Québec Centre for Biodiversity Science R Workshop Series (QCBSRworkshops, <https://github.com/QCBSRworkshops>) or the University of Edinburgh's Coding Club (Coding Club, <https://github.com/ourcodingclub>), which may be updated by an ever-evolving group of contributors over time.
+
+Using GitHub organizations as a research group or even for a handful of individuals working on a group of projects can be incredibly useful for all involved.
+GitHub organizations are relatively easy to set up, and especially easy to manage as membership to the organization changes through time.
+Not only is it a useful way to store repositories of lab-related research products, but it is also incredibly helpful for storing "living documents" that may be edited frequently, and may be linked to a lab website (that could also be generated via a repository that lives within the organization).
+The use of the "Teams" feature can allow certain groups to have varying levels of access to repositories in the organization with a select group having push access to some repositories but not others.
```

Comment:
I think it's better to write out "repository" throughout, but could be convinced otherwise.

Reply:
I also prefer repositories

Reply:
Just made issue #216 about this. I think we'll just need someone to scan through all the documents and make the switch to repository